src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy

```diff
@@ -105,6 +105,31 @@ class DockerCreateContainerFunctionalTest extends AbstractGroovyDslFunctionalTes
         result.output.contains("HWaddr 02:03:04:05:06:07")
     }
 
+    def "can create tmpfs mounts"() {
+        given:
+        String tmpFsMount = "/test-tmp-fs"
+        String containerCreationTask = """
+            task createContainer(type: DockerCreateContainer) {
+                dependsOn pullImage
+                targetImageId pullImage.getImageId()
+                cmd = ['grep', 'tmpfs $tmpFsMount', '/proc/mounts']
+                tmpfs = ['$tmpFsMount']
+                cpuset = '1'
+                labels = ["project.name": "\$project.name"]
+            }
+        """
+
+        buildFile <<
+                containerStart(containerCreationTask) <<
+                containerLogAndRemove()
+
+        when:
+        BuildResult result = build('logContainer')
+
+        then:
+        result.output.contains("tmpfs $tmpFsMount")
```

Comment:
IMO it would be better to check if tmpfs mounts are returned when inspecting container like in [this test](https://github.com/bmuschko/gradle-docker-plugin/blob/master/src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy#L24).

Reply:
sure, that's better.